src/libstd/sys/redox/backtrace/printing/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2014-2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+pub use self::imp::{foreach_symbol_fileline, resolve_symname};
+
+#[cfg(any(target_os = "macos", target_os = "ios",
+          target_os = "emscripten"))]
+#[path = "dladdr.rs"]
```

Comment:
True; I thought it might be best not to modify those files too much (I just changed `get_executable_filename` in `mod.rs`)  so they can be easily synced with the Unix version, but otherwise several things could be removed.

The duplicated code is somewhat annoying; I don't think it's avoidable though.